DependencyInjection/Configuration.php

```diff
@@ -81,6 +79,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('command')->defaultValue(null)->end()
                             ->scalarNode('connection')->defaultValue(null)->end()
                             ->scalarNode('queue')->defaultValue(null)->end()
+                            ->booleanNode('exclusive_processor_stack')->defaultValue(false)->end()
+                            ->append($this->addProcessorStack())
```

Comment:
you are missing the ``->fixXmlConfig('processor', 'processors_stack')`` on the parent node here